Networking/Networking.xcodeproj/project.pbxproj

```diff
@@ -185,6 +185,8 @@
 		209AD3C52AC19E7500825D76 /* WooPaymentsDepositsOverviewMapper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 209AD3C42AC19E7500825D76 /* WooPaymentsDepositsOverviewMapper.swift */; };
 		20D210C32B1780CE0099E517 /* deposits-overview-all.json in Resources */ = {isa = PBXBuildFile; fileRef = 20D210C22B1780CE0099E517 /* deposits-overview-all.json */; };
 		20D210C52B1788E60099E517 /* deposits-overview-all-no-default-currency.json in Resources */ = {isa = PBXBuildFile; fileRef = 20D210C42B1788E60099E517 /* deposits-overview-all-no-default-currency.json */; };
+		20F616482CF4B74600F9FA2A /* POSOrdersRemoteProtocol.swift in Sources */ = {isa = PBXBuildFile; fileRef = 20F616472CF4B74600F9FA2A /* POSOrdersRemoteProtocol.swift */; };
+		20F616492CF4B8CD00F9FA2A /* POSOrdersRemoteProtocol.swift in Sources */ = {isa = PBXBuildFile; fileRef = 20F616472CF4B74600F9FA2A /* POSOrdersRemoteProtocol.swift */; };
```

Comment:
Weird, and great catch 😊 I'll try that

Reply:
@jaclync They're not quite identical. The first ID in each line differs by a few characters:

```
20F6164   8  2CF4B   746   00F9FA2A
20F6164   9   2CF4B   8CD   00F9FA2A
```
(spacing added for clarity)

Removing and readding the file yields the same thing. I think it's because the file has to be in the watch target as well as the standard networking target, but other files in the watch target don't have the same thing. [It may be possible to remove](https://stackoverflow.com/questions/11530977/can-duplicate-file-references-in-xcodes-project-pbxproj-files-be-safely-removed), but I think it's probably safer to leave it in, because we don't tend to build the watch app directly very often.